lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -538,3 +543,182 @@ def test_move_users_to_same_cohort(self):
             ],
             verify_order=False
         )
+
+
+@patch('instructor_task.tasks_helper.DefaultStorage', new=MockDefaultStorage)
+class TestGradeReportCertificateInfo(TestReportMixin, InstructorTaskModuleTestCase):
+    """
+    Test that grade report has correct user certificate information.
+    """
+    def setUp(self):
+        super(TestGradeReportCertificateInfo, self).setUp()
+
+        self.initialize_course()
+
+        self.create_problem()
+
+        self.csv_cert_info_header = [
+            'Enrollment Track',
+            'Verification Status',
+            'Certificate Eligible',
+            'Certificate Delivered',
+            'Certificate Type'
+        ]
+
+    def create_problem(self, problem_display_name='test_problem', parent=None):
+        """
+        Create a multiple choice response problem.
+        """
+        if parent is None:
+            parent = self.problem_section
+
+        factory = MultipleChoiceResponseXMLFactory()
+        args = {'choices': [False, True, False]}
+        problem_xml = factory.build_xml(**args)
+        ItemFactory.create(
+            parent_location=parent.location,
+            parent=parent,
+            category="problem",
+            display_name=problem_display_name,
+            data=problem_xml
+        )
+
+    def generate_certificate(self, user, course_id, status, mode):
```

Comment:
I don't see a need for this method, since it is just a proxy for `GeneratedCertificateFactory.create`.